statsmodels/regression/tests/test_lme.py

```diff
@@ -808,18 +854,17 @@ def do1(reml, irf, ds_ix):
 fnames = os.listdir(rdir)
 fnames = [x for x in fnames if x.startswith("lme") and x.endswith(".csv")]
 
-import itertools
-import nose.tools
-import pytest
```

Comment:
+1 for imports at the top of the file (and I think removing a duplicated import)